City-of-Pori/pori-kada
# feature/PORI-28 main navigation

To test:

1. drush cc all
2. Make sure you have nice menu structure on your local setup with three levels of menu items.
3. The desktop navigation can fit about five first level items so you should check that you don't have more items than the design allows.
4. To fully test the menu you need to add a few e-service links. This is how its done:
Go to /admin/structure/menu/manage/main-menu/add and add a link to anywhere. On the bottom of the link add form there is accordion called "Menu item attributes". Click on that and on the "Classes" input add menu__item--e-service. After this is done save the link and make sure it is anywhere on the menu BUT on the second level of the hierarchy. Repeat this step a few times so you have nice amount of those e-service links.
5. Go to the site and test out the main menu in different browser sizes and devices.

NOTICE: This PR focuses ONLY on the main menu. If for example the language selector or the top menu with items such as "Lapsiperheet" comes on the way of the main menu, just hide them from the contexts so that they don't bother your testing. The context name is top_menu.
